modular_skyrat/modules/sec_haul/code/guns/guns.dm

```diff
@@ -305,7 +307,7 @@
 	rack_sound = 'sound/weapons/gun/pistol/rack.ogg'
 	lock_back_sound = 'sound/weapons/gun/pistol/slide_lock.ogg'
 	bolt_drop_sound = 'sound/weapons/gun/pistol/slide_drop.ogg'
-	dirt_modifier = 0.75
+	dirt_modifier = 0.3
```

Comment:
this isn't armadyne

Reply:
that was to keep it in line with the armadyne guns' new dirt balances, i'll correct the changelog

but in summary, it's a ladon with a smaller mag, so making it get dirty faster is a hard-hit for an already underused gun

Reply:
The makarov's not on the same level as armadyne guns, it's part of izhevsk's shitheap guns, you don't buy it because it's good, you buy it because it's cheap

Reply:
I don't mean to trouble you with this information, but it is around equally priced to the armadyne guns, and even, more expensive then two of them (which are better in stats)
![image](https://user-images.githubusercontent.com/62520989/172721710-47e4908e-ee99-4e8c-87f9-c6db01851108.png)
![image](https://user-images.githubusercontent.com/62520989/172721743-2750d6b2-4709-4ae6-bbb0-3e7c5b082fb4.png)